doc/source/Linux.rst

```diff
@@ -0,0 +1,71 @@
+Linux Tutorial
+==============
+
+This guide gives you a brief introduction to how volatility3 works and some demonstration on suite of plugins available from
+
+Procedure to create symbol tables for linux
+--------------------------------------------
+
+To create symbol table please refer this :ref:`symbol-tables:Mac or Linux symbol tables`.
+You can also find some ISF files from this website `Linux ISF Server <https://isf-server.techanarchy.net/>`_ Which is built and maintained by `kevthehermit <https://twitter.com/kevthehermit>`_.
+
+Using plugins
+-------------
+
+The following is the syntax to run volatility tool.
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f <path to memory image> plugin_name plugin_option
+
+List of Plugins
+----------------
+
+Following are the list of linux plugins available for volatility3. More plugins will be available on future releases.
```

Comment:
I'd kind of prefer to just show users how they'd find this information from the tool, rather than including a list (that will get out of date) in the documentation.